src/assets/styleJs/events/createEvent.ts

```diff
@@ -0,0 +1,224 @@
+import {makeStyles} from "@mui/styles";
+import {borderRedius, color, font, margin, padding} from "../../style/globalVariables";
+
+const useCreateEventStyles = makeStyles({
+  rootCreateEvent: {
+    height: "100%"
+  },
+
+  eventTitleBox: {
+    display: "flex",
+    alignItems: "center",
+    width: "45%",
+    marginBottom: margin.marginBottom40
+  },
+  eventInfoBoxEvaluator: {
+    display: "flex",
+    alignItems: "center",
+    width: "40%",
+    marginBottom: margin.marginBottom25
+  },
+  eventInfoBoxEvaluatee: {
+    display: "flex",
+    alignItems: "center",
+    width: "40%",
+    marginBottom: margin.marginBottom25,
+  },
```

Comment:
same thing in 2 places use the same class for both

Reply:
done

Reply:
not fixed?